pages/linux/radeontop.md

```diff
@@ -5,16 +5,16 @@
 
 - Show the utilisation of the default AMD GPU:
 
-`sudo radeontop`
+`radeontop`
 
 - Enable colourised output:
```

Comment:
Should we be using US or UK spellings (or does it not matter)? :thinking:

Reply:
We're supposed to prefer US spelling but, not go out of our way to change all the pages.  https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#default-language-for-newly-added-pages

Reply:
I'd probably change this to `coloured` anyway, regardless of the spelling.